diff --git a/node_modules/@mui/system/esm/createTheme/shape.d.ts b/node_modules/@mui/system/esm/createTheme/shape.d.ts
index 8bb933f..8695988 100644
--- a/node_modules/@mui/system/esm/createTheme/shape.d.ts
+++ b/node_modules/@mui/system/esm/createTheme/shape.d.ts
@@ -1,5 +1,5 @@
 export interface Shape {
-  borderRadius: number | string;
+  borderRadius: number;
 }
 export type ShapeOptions = Partial<Shape>;
 declare const shape: Shape;
